src/inference/src/cpp_interfaces/interface/ie_iinfer_request_internal.cpp

```diff
@@ -164,14 +166,20 @@ Blob::Ptr IInferRequestInternal::GetBlob(const std::string& name) {
     DataPtr foundOutput;
     const SizeVector oneVector = {1};
     if (findInputAndOutputBlobByName(name, foundInput, foundOutput)) {
+        IE_SUPPRESS_DEPRECATED_START
+        const bool isInputDynamic = foundInput && foundInput->getInputData()->isDynamic();
+        IE_SUPPRESS_DEPRECATED_END
```

Comment:
don't use deprecated API

Reply:
@ilya, can you suggest which API I should use for dynamic case? I was referring to https://github.com/openvinotoolkit/openvino/blob/a77d6cfe36b49a90c9f49d35abe548dbfdd2e0d2/src/inference/src/cpp_interfaces/interface/ie_iinfer_request_internal.cpp#L103

Reply:
https://github.com/openvinotoolkit/openvino/blob/master/src/inference/dev_api/cpp_interfaces/interface/ie_iinfer_request_internal.hpp#L202-L203